spec/ffprober/parser_spec.rb

```diff
@@ -26,6 +26,24 @@ def shared_specs
     end
   end
 
+    describe 'subtitle_streams' do
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.